### PR TITLE
Aws o terraform

### PR DESCRIPTION
**Pull request by SumoSourabh**
Adding AWS Observability Terraform script to deploy Cloudformation template using Terraform script.

### CODE REVIEW

File: aws-observability-terraform/versions.tf [hunk 1]
```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+    }
+    null = {
+      source = "hashicorp/null"
+    }
+    sumologic = {
+      source = "terraform-providers/sumologic"
```
Comment (arunpatyal) [inline aws-observability-terraform/versions.tf:10]: Better to lock the lowest version for all providers i.e. sumo, aws and null

File: aws-observability-terraform/main_variables.auto.tfvars [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## AWS CloudFormation Stack Name. Provide a unique AWS CloudFormation Stack Name.
+# Please replace <YOUR STACK NAME> (including brackets) with stack name.
+CloudFormationStackName = "<YOUR STACK NAME>"
+
+# For more details on parameters, Please visit - https://help.sumologic.com/Solutions/AWS_Observability_Solution/03_Set_Up_the_AWS_Observability_Solution#Configuration_prompts_and_input
+
+####### BELOW ARE REQUIRED PARAMETERS FOR CLOUDFORMATION STACK #######
+# Visit - https://help.sumologic.com/Solutions/AWS_Observability_Solution/03_Set_Up_the_AWS_Observability_Solution#sumo-logic-access-configuration-required
+Section1aSumoLogicDeployment = "<YOUR SUMO DEPLOYMENT>" # Please replace <YOUR SUMO DEPLOYMENT> (including brackets) with au, ca, de, eu, jp, us2, in, fed or us1.
```
Comment (arunpatyal) [inline aws-observability-terraform/main_variables.auto.tfvars:14]: Why the variables are named as SectionXVariableName. Is it helpful to customers in anyway?

**Review by arunpatyal — CHANGES_REQUESTED** [review]
Run terraform fmt on all files i.e. in the code folder, it will format the code.

File: aws-observability-terraform/versions.tf (hunk 1 above)
Comment (SumoSourabh) [reply aws-observability-terraform/versions.tf]: Done.

File: aws-observability-terraform/main_variables.auto.tfvars (hunk 2 above)
Comment (SumoSourabh) [reply aws-observability-terraform/main_variables.auto.tfvars]: This is done in order to keep sync between Cloudformation template parameters and terraform parameters.

**Comment by SumoSourabh** [comment]
> Run terraform fmt on all files i.e. in the code folder, it will format the code.

Done

**Review by arunpatyal — APPROVED** [review]
LGTM

**Pull request merged**